api/src/main/kotlin/com/gatehill/corebot/config/model/ActionConfig.kt

```diff
@@ -11,22 +11,28 @@ class ActionConfig(val template: String,
                    val jobId: String,
                    name: String?,
                    options: Map<String, OptionConfig>?,
-                   tags: List<String>?, driver: String?) {
+                   tags: List<String>?, driver: String?,
+                   showJobOutput: String?, showJobInfo: String?) {
 
     val name: String
     val options: Map<String, OptionConfig>
     val tags: List<String>
     val driver: String
+    val showJobOutput: String
+    val showJobInfo: String
 
     init {
         this.name = name ?: ""
         this.options = options ?: emptyMap()
         this.tags = tags ?: emptyList()
         this.driver = driver ?: defaultDriver
+        this.showJobOutput = showJobOutput ?: "false"
+        this.showJobInfo = showJobInfo ?: "true"
```

Comment:
We should call this something like 'job outcome', as 'info' feels a bit too general.